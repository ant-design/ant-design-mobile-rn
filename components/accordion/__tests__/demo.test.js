import rnDemoTest from '../../../tests/shared/demoTest'

rnDemoTest('accordion')
