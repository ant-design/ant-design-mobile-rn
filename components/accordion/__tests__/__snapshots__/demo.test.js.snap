// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/accordion/demo/basic.tsx correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 10,
      "marginTop": 80,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "borderTopColor": "#dddddd",
          "borderTopWidth": 0.5,
        },
        undefined,
      ]
    }
  >
    <View>
      <View
        style={Object {}}
      >
        <
          onPress={[Function]}
          underlayColor="transparent"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "borderBottomColor": "#dddddd",
                  "borderBottomWidth": 0.5,
                  "flexDirection": "row",
                  "paddingLeft": 15,
                  "paddingRight": 30,
                },
                Object {},
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flex": 1,
                  "flexDirection": "row",
                  "height": 44,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000000",
                    "fontSize": 17,
                  }
                }
              >
                Title 1
              </Text>
            </View>
            <View
              style={
                Object {
                  "color": "#cccccc",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontFamily": "antoutline",
                      "fontSize": 22,
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {
                      "color": "#cccccc",
                    },
                  ]
                }
              >
                
              </Text>
            </View>
          </View>
        </>
        <View
          pointerEvents="auto"
          style={false}
        >
          <View
            forwardedRef={[Function]}
            onLayout={[Function]}
            style={Object {}}
          >
            <View>
              <View
                style={
                  Object {
                    "backgroundColor": "#ffffff",
                    "borderTopColor": "#dddddd",
                    "borderTopWidth": 0.5,
                  }
                }
              >
                <
                  activeOpacity={1}
                  delayLongPress={500}
                  onLongPress={[Function]}
                >
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "#ffffff",
                          "flexDirection": "row",
                          "flexGrow": 1,
                          "paddingLeft": 15,
                        },
                        undefined,
                      ]
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "borderBottomColor": "#dddddd",
                            "borderBottomWidth": 0.5,
                            "flex": 1,
                            "flexDirection": "row",
                            "minHeight": 44,
                            "paddingRight": 15,
                            "paddingVertical": 6,
                          },
                          false,
                          false,
                        ]
                      }
                    >
                      <View
                        style={
                          Array [
                            Object {
                              "flex": 1,
                              "flexDirection": "column",
                            },
                          ]
                        }
                      >
                        <Text
                          numberOfLines={1}
                          style={
                            Array [
                              Object {
                                "color": "#000000",
                                "fontSize": 17,
                                "textAlignVertical": "center",
                              },
                            ]
                          }
                        >
                          Content 1
                        </Text>
                      </View>
                    </View>
                  </View>
                </>
                <
                  activeOpacity={1}
                  delayLongPress={500}
                  onLongPress={[Function]}
                >
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "#ffffff",
                          "flexDirection": "row",
                          "flexGrow": 1,
                          "paddingLeft": 15,
                        },
                        undefined,
                      ]
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "borderBottomColor": "#dddddd",
                            "borderBottomWidth": 0.5,
                            "flex": 1,
                            "flexDirection": "row",
                            "minHeight": 44,
                            "paddingRight": 15,
                            "paddingVertical": 6,
                          },
                          false,
                          false,
                        ]
                      }
                    >
                      <View
                        style={
                          Array [
                            Object {
                              "flex": 1,
                              "flexDirection": "column",
                            },
                          ]
                        }
                      >
                        <Text
                          numberOfLines={1}
                          style={
                            Array [
                              Object {
                                "color": "#000000",
                                "fontSize": 17,
                                "textAlignVertical": "center",
                              },
                            ]
                          }
                        >
                          Content 2
                        </Text>
                      </View>
                    </View>
                  </View>
                </>
                <
                  activeOpacity={1}
                  delayLongPress={500}
                  onLongPress={[Function]}
                >
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "#ffffff",
                          "flexDirection": "row",
                          "flexGrow": 1,
                          "paddingLeft": 15,
                        },
                        undefined,
                      ]
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "borderBottomColor": "#dddddd",
                            "borderBottomWidth": 0.5,
                            "flex": 1,
                            "flexDirection": "row",
                            "minHeight": 44,
                            "paddingRight": 15,
                            "paddingVertical": 6,
                          },
                          false,
                          false,
                        ]
                      }
                    >
                      <View
                        style={
                          Array [
                            Object {
                              "flex": 1,
                              "flexDirection": "column",
                            },
                          ]
                        }
                      >
                        <Text
                          numberOfLines={1}
                          style={
                            Array [
                              Object {
                                "color": "#000000",
                                "fontSize": 17,
                                "textAlignVertical": "center",
                              },
                            ]
                          }
                        >
                          Content 3
                        </Text>
                      </View>
                    </View>
                  </View>
                </>
                <View
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#ffffff",
                        "borderBottomColor": "#dddddd",
                        "borderBottomWidth": 0.5,
                        "bottom": 0,
                        "height": 1,
                        "left": 0,
                        "position": "absolute",
                        "right": 0,
                      },
                    ]
                  }
                />
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        style={Object {}}
      >
        <
          onPress={[Function]}
          underlayColor="transparent"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "borderBottomColor": "#dddddd",
                  "borderBottomWidth": 0.5,
                  "flexDirection": "row",
                  "paddingLeft": 15,
                  "paddingRight": 30,
                },
                Object {},
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flex": 1,
                  "flexDirection": "row",
                  "height": 44,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000000",
                    "fontSize": 17,
                  }
                }
              >
                Title 2
              </Text>
            </View>
            <View
              style={
                Object {
                  "color": "#cccccc",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontFamily": "antoutline",
                      "fontSize": 22,
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {
                      "color": "#cccccc",
                    },
                  ]
                }
              >
                
              </Text>
            </View>
          </View>
        </>
        <View
          pointerEvents="none"
          style={
            Object {
              "height": 0,
              "overflow": "hidden",
            }
          }
        >
          <View
            forwardedRef={[Function]}
            onLayout={[Function]}
            style={Object {}}
          >
            <View
              style={
                Object {
                  "borderBottomColor": "#dddddd",
                  "borderBottomWidth": 0.5,
                  "paddingHorizontal": 8,
                  "paddingVertical": 9,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333",
                    "fontSize": 15,
                  }
                }
              >
                this is panel content2 or other
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        style={Object {}}
      >
        <
          onPress={[Function]}
          underlayColor="transparent"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "borderBottomColor": "#dddddd",
                  "borderBottomWidth": 0.5,
                  "flexDirection": "row",
                  "paddingLeft": 15,
                  "paddingRight": 30,
                },
                Object {},
              ]
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "flex": 1,
                  "flexDirection": "row",
                  "height": 44,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000000",
                    "fontSize": 17,
                  }
                }
              >
                Title 3
              </Text>
            </View>
            <View
              style={
                Object {
                  "color": "#cccccc",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontFamily": "antoutline",
                      "fontSize": 22,
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {
                      "color": "#cccccc",
                    },
                  ]
                }
              >
                
              </Text>
            </View>
          </View>
        </>
        <View
          pointerEvents="auto"
          style={false}
        >
          <View
            forwardedRef={[Function]}
            onLayout={[Function]}
            style={Object {}}
          >
            <View
              style={
                Object {
                  "borderBottomColor": "#dddddd",
                  "borderBottomWidth": 0.5,
                  "paddingHorizontal": 8,
                  "paddingVertical": 9,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333",
                    "fontSize": 15,
                  }
                }
              >
                Text text text text text text text text text text text text text text text
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
